src/cmd/cli/command/compose.go

```diff
@@ -5,19 +5,50 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"slices"
 	"strings"
 	"time"
 
 	"github.com/AlecAivazis/survey/v2"
+	"github.com/DefangLabs/defang/src/pkg"
 	"github.com/DefangLabs/defang/src/pkg/cli"
 	cliClient "github.com/DefangLabs/defang/src/pkg/cli/client"
 	"github.com/DefangLabs/defang/src/pkg/term"
 	"github.com/DefangLabs/defang/src/pkg/types"
 	defangv1 "github.com/DefangLabs/defang/src/protos/io/defang/v1"
 	"github.com/bufbuild/connect-go"
+	compose "github.com/compose-spec/compose-go/v2/types"
 	"github.com/spf13/cobra"
 )
 
+var managed_resources = []string{"x-defang-redis", "x-defang-postgres", "x-defang-static-files"}
+
+func isManagedResource(extension string) bool {
+	return slices.Contains(managed_resources, extension)
+}
+
+// return a list of resources not being depended on by other services
+func GetUnreferencedManagedResources(project *compose.Project) []string {
```

Comment:
get a list of managed resource where other services does not have an explicit depends-on them. we no longer assume that just because there are unmanaged service we can use their health check for assumer the managed service is up unless there is a depend-on defiend.

Reply:
we don't currently use `depends_on` at all, so what does this test?